### DISCUSSION TITLE
Server Status Offline in browser when using Immich behind Cloudflare Tunnel

### DISCUSSION THREAD

**Discussion by islandbeachbum** [immich-app/immich#9528] [Feature Request]
### The bug

On the mobile app the Server Version is showing normally, in the browser its showing status "Offline" and version "Unknown", while still working perfectly fine. Seems to be because Cloudflare (Argo) Tunnels block wss requests. 

Would be nice if there were a regular https fallback to the wss request to check the server status from the browser.

Request header of the failing request:
![Screenshot from 2024-05-16 09-49-57](https://github.com/immich-app/immich/assets/88225742/d1f2992d-2ae3-4d9e-92bc-b4078b816ac4)

Request header of the successful initial request when loading the same page:
![Screenshot from 2024-05-16 09-49-22](https://github.com/immich-app/immich/assets/88225742/9bee0e56-ced4-4327-a265-e5559618ff12)

### The OS that Immich Server is running on

Nixos 23.11

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

v1.105.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Default
```

### Your .env content

```Shell
Default
```

### Reproduction steps

```bash
1 - Put Immich behind Cloudflare (Argo) Tunnel
2 - Visit Immich URL in browser (firefox)
3 - Check Server Status and Version in the left-bottom corner
4 - Showing Offline and Unknown
5 - Route DNS directly to server IP, not via Cloudflare Tunnel
6 - Check Server Status and Version in the left-bottom corner
7 - Showing as normal
```

**Comment by jrasm91** [comment]
This is working as designed, but we could potentially implement a fallback. Although, a decent amount of functionality assume a websocket exists, including picking a new person thumbnail, uploading a profile picture, uploading and deleting assets, etc. You would have to refresh the page manually to see the changes without a websocket connection.

**Reply by mmomjian** [reply]
My .02 is that we should not implement a fallback, because the current implementation makes it very obvious that there is an issue with the proxy server. If we fix this, we will just get more vague bug reports with issues of uploads/downloads. People regularly ask about this issue here and on Discord and it's an easy way to identify that WSS is not being proxied.

**Reply by bo0tzz** [reply]
Note, we deliberately removed the polling fallback in #6564

**Reply by jrasm91** [reply]
We could implement polling the status endpoint not the web socket connection. This is how it worked before.
